OpenImis.ModulesV2/InsureeModule/InsureeModule.cs

```diff
@@ -41,7 +41,7 @@ public IPolicyLogic GetPolicyLogic()
         {
             if (_policyLogic == null)
             {
-                _policyLogic = new PolicyLogic(_configuration);
+                _policyLogic = new PolicyLogic(_configuration, _hostingEnvironment);
```

Comment:
Please create another module called Policy which will contain Policy and PolicyRenewal. Move this logic to the Policy Module and cal it PolicyRenewalLogic as it deals with renewals. If not it will be misinterpreted.